app/models/team.rb

```diff
@@ -256,5 +261,12 @@ def self.create_team_and_node(id)
     team
   end
 
-  # REFACTOR END:: class methods import export moved from course_team & assignment_team to here
+  def self.get_team(participant)
```

Comment:
I don't think you need to write a method to get the team info from a participant. 
You can try `Team.find_by(TeamsUser.team_id(participant.parent_id, participant.user_id) ||= 0)`
Also, calling a model method from view is not a good idea. It will be better to move the method to corresponding helper file if you want to keep this method/